packages/openapi-parser/src/utils/traverse.ts

```diff
@@ -5,25 +5,26 @@ import type { AnyObject } from '../types'
  */
 export function traverse(
   specification: AnyObject,
-  transform: (specification: AnyObject) => AnyObject,
+  transform: (specification: AnyObject, path: string[]) => AnyObject,
+  path: string[] = [],
```

Comment:
The traverse function is also used by 
- `filter`
- `getListOfReferences`
- `upgradeFromTwoToThree`

We should update those or make the path argument optional

Reply:
True!